Entities/Vehicles/Catapult/Catapult.as

```diff
@@ -27,20 +28,19 @@ void onInit(CBlob@ this)
 		return;
 	}
 
-	v.max_charge_time = 90;
-	v.max_cooldown_time = cooldown_time;
-
-	Vehicle_SetupWeapon(this, v,
+	Vehicle_AddAmmo(this, v,
 	                    cooldown_time, // fire delay (ticks)
 	                    5, // fire bullets amount
-	                    getMagAttachmentPoint(this).offset, // fire position offset
+	                    2, // fire cost
```

Comment:
I don't understand this. Couldn't you just double the fire amount to achieve the same thing?

Reply:
The catapult fires 5 cata_rocks at the cost of 2 mat_stone per bullet, making it 10 mat_stone total for 5 cata_rocks. Making the cost be 1 and the fire bullet amount be 10 would double the cata_rocks amount fired